egs/wsj/s5/steps/nnet3/decode_score_fusion.sh

```diff
@@ -140,11 +145,13 @@ for i in `seq 0 $[num_sys-1]`; do
   
   # check that they have the same frame-subsampling-factor
   if [ $frame_subsampling_factor -ne `cat $srcdir/frame_subsampling_factor` ]; then
-    echo "$0 frame_subsampling_factor must be the same."
+    echo "$0 frame_subsampling_factor must be the same.\\"
+    echo "Default:$frame_subsampling_factor \\"
+    echo "In $srcdir:`cat $srcdir/frame_subsampling_factor`"
     exit 0;
   fi
```

Comment:
thanks.  would you mind also changing it to print a warning if frame_subsampling_factor is 3 and acwt is 0.1?

Reply:
Thanks for the advice.
I want to ask you a question regarding this warning.

If true ((frame_subsampling_factor=3) && (acwt != 1.0) || (post_decode_acwt) != 10.0) 
Should I exit this script or just print WARNING ?